From 6f639c450c513c44b6eb7ddfd31e207ca83bcd39 Mon Sep 17 00:00:00 2001
From: Cyril Brulebois <kibi@debian.org>
Date: Sat, 31 Dec 2011 19:35:17 +0100
Subject: [PATCH:xf86-input-evdev 1/2] configure.ac: Fix udev/libudev
 dependency.

In 683a55e504f4fc2d1c847c54986439a0c61b2f20, a dependency on libudev was
added, but documented in configure.ac as a dependency on udev (which
also happens to ship a pkg-config file).

Signed-off-by: Cyril Brulebois <kibi@debian.org>
---
 configure.ac |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/configure.ac b/configure.ac
index 57690c4..47f70b4 100644
--- a/configure.ac
+++ b/configure.ac
@@ -46,7 +46,7 @@ XORG_DEFAULT_OPTIONS
 
 # Obtain compiler/linker options from server and required extensions
 PKG_CHECK_MODULES(XORG, [xorg-server >= 1.10] xproto inputproto)
-PKG_CHECK_MODULES(UDEV, udev)
+PKG_CHECK_MODULES(UDEV, libudev)
 
 PKG_CHECK_MODULES(XI22, [inputproto >= 2.1.99.3] [xorg-server >= 1.11.99.901], HAVE_XI22="yes", HAVE_XI22="no")
 
-- 
1.7.7.3

